GDJS/Runtime/layer.js

```diff
@@ -216,6 +216,24 @@ gdjs.Layer.prototype.setEffectParameter = function(name, parameterIndex, value)
     return this._renderer.setEffectParameter(name, parameterIndex, value);
 };
 
+/**
+ * Enable an effect.
+ * @param {string} filter The name of the filter effect
```

Comment:
"The name of the effect to enable or disable."

The term "filter" is Pixi.js specific, in GDevelop I think it's better to keep the metaphor of the "effect" everywhere.